src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -736,7 +736,7 @@ public void AcceptV6BoundToAnyV6_Success()
         }
 
         [Fact]
-        [ActiveIssue(5832, PlatformID.AnyUnix)]
+        [PlatformSpecific(PlatformID.Windows)] // https://github.com/dotnet/corefx/issues/5832
```

Comment:
The linked issue makes it sound like this isn't safe on Windows either, that it could result in AVs.  Are these tests ok for Windows, or should they be deleted entirely?